apps/paf/index.js

```diff
@@ -305,7 +531,7 @@ module.exports = {
         'personAddFirstName',
```

Comment:
HOF follows kebab style for variable names

Reply:
I have only used camelcase for the aggregator fields similar to the other services that use the aggregator behaviour. When I tried to change the format, things were breaking so I will look into that as a separate task as it might have been setup this way. This is only for the person section though so shouldn't affect this pr.